app/models/user.rb

```diff
@@ -60,6 +69,27 @@ def send_password_reset_email
   def password_reset_expired?
     reset_sent_at < 2.hours.ago
   end
+
+  def feed
+    following_ids = "SELECT followed_id FROM relationships
+      WHERE  follower_id = :user_id"
+    Micropost.where "user_id IN (#{following_ids})
```

Comment:
cái mệnh đề: where "user_id IN (#{following_ids}) ...  này a chuyển sang viết scope nhé
scope này viết ở model Micropost nhé